CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+#### 15.0.0-alpha.0 (2023-08-19)
```

Comment:
Hmm, should we make a regular PR to reduce the number to 13, such that the CI bumps it to 14?

Reply:
Yes I think we need to and also remove the v15 git tag